interpreter/Makefile

```diff
@@ -88,6 +88,13 @@ install:
 	dune build -p $(NAME) @install
 	dune install
 
+.PHONY: opam-release
+
+opam-release/%:
+	git tag opam-$*
+	git push --tags
+	opam publish
```

Comment:
IIRC it's going to ask.

Reply:
I just tried opam-publish with all variations and available options, but it always said "Not enough information to publish", even when I set the version explicitly. My suspicion is that it bakes in assumptions like package-name = repo-name.

The implementation details don't really matter to me, but this target should continue working.

Reply:
Indeed, it breaks also `dune-release`. This is because we don't have `wasm.opam` at the root of the repository...
Should I move `dune-project` and `wasm.opam` at the root (I know you don't like it) ? Otherwise we can keep the previous version.

Reply:
Also note that having them at the root of the repository will make `opam pin https://github.com/WebAssembly/spec` to work properly.

Reply:
By root you mean the root of the repo? I'm afraid that wouldn't be adequate, since the interpreter is only a secondary subcomponent of this project that shouldn't affect the root dir. It would also slightly break path-dependent CI.

Would perhaps opams-publish's --packages-directory=interpreter option work in this case, by any chance?

Reply:
That's unfortunate, but thanks for poking. Let's revert to what was there before then. Maybe opam will support it at some point in the future.

(CI is set up per toplevel directory, i.e., interpreter builds only run when something under interpreter/ changes. Moving dune and opam files out of there would then fail to capture them. Although that could probably be hacked.)